src/newcharacter.cpp

```diff
@@ -67,9 +67,13 @@ void save_template(player *u);
 
 bool lcmatch(const std::string &str, const std::string &findstr); // ui.cpp
 
-void Character::pick_name()
+void Character::pick_name(bool bUseDefault)
```

Comment:
The "pick random name" command in the character creation doesn't work with this. It calls `pick_name` without parameters, therefor the default name is always selected, never a random name.